test/unit/DCAStrategies/dca-strategies-management-handler.spec.ts

```diff
@@ -27,6 +28,18 @@ contract('DCAStrategiesManagementHandler', () => {
     await snapshot.revert(snapshotId);
   });
 
+  function generateRandomAddress() {
+    return ethers.utils.getAddress(ethers.utils.hexlify(ethers.utils.randomBytes(20)));
+  }
+
+  function compareTokens(arrayA: IDCAStrategies.ShareOfTokenStruct[], arrayB: IDCAStrategies.ShareOfTokenStruct[]) {
```

Comment:
Makes sense to have this util function, but we usually try to leave it after the "important code". In production code, we usually do:
1. External/public
2. Internal/private

In tests, we usually do:
1. Tests
2. Utils

Does it make sense?